aws-iot-scheduledaudit/aws-iot-scheduledaudit.json

```diff
@@ -0,0 +1,134 @@
+{
+  "typeName": "AWS::IoT::ScheduledAudit",
+  "description": "Scheduled audits can be used to specify the checks you want to perform during an audit and how often the audit should be run.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-iot.git",
+  "definitions": {
+    "Tag": {
+      "description": "A key-value pair to associate with a resource.",
+      "type": "object",
```

Comment:
(Fixed this recently in my PR as well) all entries with type=object should have `"additionalProperties": false`

Reply:
Added